spec/io_spec.rb

```diff
@@ -26,7 +26,7 @@
 #
 require "tmpdir" # Used to avoid cluttering the repository.
 require_relative 'spec_helper'
-require_relative "../lib/nmatrix"
+require "./lib/nmatrix"
```

Comment:
Cool. BLAS spec is OK. Looking at the error messages; I think the error is in C header file. Will correct it. 
I will configure JRuby tests in Travis :)
However, since 100% tests don't succeed, I think I can keep it as a future task.